logstash-core/src/main/java/org/logstash/ext/JrubyEventExtLibrary.java

```diff
@@ -96,7 +96,7 @@ public IRubyObject ruby_set_field(ThreadContext context, RubyString reference, I
                 }
                 this.event.setTimestamp(((JrubyTimestampExtLibrary.RubyTimestamp)value).getTimestamp());
             } else {
-                this.event.setField(r, Valuefier.convert(value));
+                this.event.setField(r, safeValueifierConvert(value));
```

Comment:
In my testing, illegal field references are always caught above in the `extractFieldReference` call on line 92. Is there a scenario I'm missing?